cgmanifest.json

```diff
@@ -6020,6 +6640,36 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "recordproto",
```

Comment:
The `redis` package above should be behind this one.

Reply:
fixed